code/game/machinery/gulag_processor.dm

```diff
@@ -182,7 +182,7 @@ GLOBAL_VAR_INIT(gulag_required_items, typecacheof(list(
 
 	open_machine()
 	prisoner.Paralyze(stun_duration)
-	if(!prisoner.handcuffed)
+	if((prisoner.get_num_arms(FALSE) >= 2 || prisoner.get_arm_ignore()))
 		prisoner.handcuffed = new /obj/item/restraints/handcuffs/cable/zipties/used(prisoner)
```

Comment:
please include the !prisoner.handcuffed check still
otherwise this can overwrite proper handcuffs with ziptie handcuffs